doc/installation/system/securitymodule.rst

```diff
@@ -134,7 +134,10 @@ To use the correct key in this slot you can either specify the key by providing
 ``PI_HSM_MODULE_KEYID`` with the integer id of the key or
 ``PI_HSM_MODULE_KEYLABEL``  with the descriptive label of the key.
 
-The ``PI_HSM_MODULE_TIMEOUT`` can be used to define an integer value for a HSM lock timeout.
+The ``PI_HSM_MODULE_TIMEOUT`` can be used to define an integer value for a HSM lock timeout. The default is 15 seconds.
+
+Using the key ``PI_HSM_MODULE_LOCK_DIR`` you can define a different locking directory.
```

Comment:
You might want to explain that the directory itself is the lock and should not exist before starting the service, otherwise people might create the directory beforehand.